examples/loadHttp.md

```diff
@@ -0,0 +1,25 @@
+# load file from http
+
+Resource and patch files could be loaded from http
+
+<!-- @loadHttp -->
+```sh
+DEMO_HOME=$(mktemp -d)
+
+cat <<EOF >$DEMO_HOME/kustomization.yaml
+resources:
+- https://raw.githubusercontent.com/kubernetes-sigs/kustomize/master/examples/helloWorld/configMap.yaml
+EOF
+```
+
+<!-- @loadHttp -->
+```sh
+test 1 == \
+  $(kustomize build $DEMO_HOME | grep "Good Morning!" | wc -l); \
+  echo $?
+```
+
+Kustomize will try loading resource as a file either from local or http. If it
+fails, try to load it as a directory or git repository.
```

Comment:
What does "load it as a directory or git repository" mean?

Reply:
Example of load as a directory or git repository:
```yaml
resources:
- ../base
- github.com/kustless/kustomize-examples
```
It is not possible to tell if it is a file before trying to load.

Reply:
So that example is now valid? The [documentation](https://github.com/yujunz/kustomize/blob/382425d974c9d2f30816fbd7d3794aebf62ecbdf/examples/loadHttp.md) only mentions loading via `http[s]://` urls.

Reply:
The example should be valid at 300dd108d5c505ff436f77138f457231566990b4 but might be broken on 0b1ad031a9e158a51498c7026e1ed2768c4806db , see #2264 .

`github.com/kustless/kustomize-examples` is supported by the original git cloner. http loader is for single yaml file only as given in the example. It does not apply for git repository or http archive. When failed to load file, it will fallback to original directory loader and git cloner.

#2169 try to solve the remote get problem with go-getter but it was merged by accident before completely tested...